source/base/data_out_base.inst.in

```diff
@@ -14,144 +14,172 @@
-                  const std::vector<std::string>          &data_names,
-                  const std::vector<std::tuple<unsigned int, unsigned int, std::string> > &vector_data_ranges,
-                  const PovrayFlags                          &flags,
-                  std::ostream                            &out);
-
-    template
-    void
-    write_eps (const std::vector<Patch<deal_II_dimension,deal_II_space_dimension> > &patches,
-               const std::vector<std::string>          &data_names,
-               const std::vector<std::tuple<unsigned int, unsigned int, std::string> > &vector_data_ranges,
-               const EpsFlags                          &flags,
-               std::ostream                            &out);
-
-    template
-    void
-    write_gmv (const std::vector<Patch<deal_II_dimension,deal_II_space_dimension> > &patches,
-               const std::vector<std::string>          &data_names,
-               const std::vector<std::tuple<unsigned int, unsigned int, std::string> > &vector_data_ranges,
-               const GmvFlags                          &flags,
-               std::ostream                            &out);
-
-    template
-    void
-    write_tecplot (const std::vector<Patch<deal_II_dimension,deal_II_space_dimension> > &patches,
-                   const std::vector<std::string>          &data_names,
-                   const std::vector<std::tuple<unsigned int, unsigned int, std::string> > &vector_data_ranges,
-                   const TecplotFlags                          &flags,
-                   std::ostream                            &out);
-
-    template
-    void
-    write_tecplot_binary (const std::vector<Patch<deal_II_dimension,deal_II_space_dimension> > &patches,
-                          const std::vector<std::string>          &data_names,
-                          const std::vector<std::tuple<unsigned int, unsigned int, std::string> > &vector_data_ranges,
-                          const TecplotFlags                          &flags,
-                          std::ostream                            &out);
-
-#if deal_II_space_dimension >1
-    template
-    void
-    write_svg (const std::vector<Patch<deal_II_dimension,deal_II_space_dimension> > &patches,
-               const std::vector<std::string>          &data_names,
-               const std::vector<std::tuple<unsigned int, unsigned int, std::string> > &vector_data_ranges,
-               const SvgFlags                          &flags,
-               std::ostream                            &out);
+      template void
+      write_vtk(
+        const std::vector<Patch<deal_II_dimension, deal_II_space_dimension>>
+          &                             patches,
+        const std::vector<std::string> &data_names,
+        const std::vector<std::tuple<unsigned int, unsigned int, std::string>>
+          &             vector_data_ranges,
+        const VtkFlags &flags,
```

Comment:
I am not a huge fan of this right alignment. It is quite buggy :-/

Reply:
Yes, basically everything about this change looks wrong :-(

Reply:
Shall we merge this as-is and for now switch to left alignment?

Reply:
I don't think I have an opinion on what's better. I liked our old indentation in this regard, both as far as the placement of the `&` is concerned as well as putting the arguments on individual lines and keeping them with the opening parenthesis. I don't like any of this, but I don't know that I would like it better if we chose any other clang-format option.

Reply:
Ok, let’s merge this anyway as right or left alignment is a bit off-topic in this PR